src/js/chat.js

```diff
@@ -505,7 +501,7 @@ sock.onmessage = function (e) {
             case "332":
                 // Topic, display?
                 break;
-            // Part and quit both need to be handled the same way in our case - a user left the room
+                // Part and quit both need to be handled the same way in our case - a user left the room
```

Comment:
Actually, I was inconsistent with the commenting in that switch statement. I think most of the time I put it inside. I kinda feel like it makes more sense inside. Thoughts?